global.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
the issue arises is when you want to use `dotnet <command>` in the terminal, like if you want to build, restore, or run only one particular project